global/global-constants.js

```diff
@@ -55,6 +55,11 @@ const POPUP_HEIGHT_L = 100;
 const SINGLE_LINE = 50;
 const DOUBLE_LINE = 60;
 const WEATHER_MOON_LINE = 95
+const tutorial_styles = {
+  singleLine: [{backgroundColor: primary_color, height: SINGLE_LINE}],
+  doubleLine: [{backgroundColor: primary_color, height: DOUBLE_LINE}],
```

Comment:
Even with these tutorial sizing options, the tutorial text still overflows. I feel this approach to tutorial heights isn't the best, as there could be languages with much larger text density than english. Could there be a way of dynamically setting the height?

I changed these values (SINGLE_LINE & DOUBLE_LINE), and they both seem to not have an affect on the tutorial boxes, on my end at least.
![Screenshot_20240426_104732_Expo_Go](https://cdn.discordapp.com/attachments/1223039100781727859/1233440954065621033/Screenshot_20240426_111125_Expo_Go.jpg?ex=662d1ae7&is=662bc967&hm=323b89c735bde5c22e9312128fe8774d6126d3e73bb6394b87251ae7c096226e&)

Reply:
I couldn't find anything about text wrap/flexbox type stuff for tooltips. It's hard to find styling information in general about this dependency. For now I just increased the width

Reply:
Additional note: the LuresForm tutorials have not been adjusted yet